README.md

```diff
@@ -61,7 +61,10 @@ var slackbot = new SlackBot({
 
 ## Responding to Buttons with Actions
 
+Before you can have `Buttons with Actions` you will need to setup a [SlackApp][https://api.slack.com/slack-apps], note you do not need to publicly publish your SlackApp to utilize the Slack buttons.
```

Comment:
Think you just need to fix the link part of the markdown here to be `(https...)` instead of `[https...]` and we're good to go!